ghjk.ts

```diff
@@ -69,13 +76,147 @@ if (!Deno.env.has("NO_PYTHON")) {
   }
 }
 
-if (!Deno.env.has("CI") && !Deno.env.has("OCI")) {
+if (inDev()) {
   ghjk.install(
     ports.act({}),
-    ports.whiz({}),
+    ports.cargobi({ crateName: "whiz" }),
   );
 }
 
+const allowedPortDeps = [...ghjk.stdDeps({ enableRuntimes: true })];
+
 export const secureConfig = ghjk.secureConfig({
-  allowedPortDeps: [...ghjk.stdDeps({ enableRuntimes: true })],
+  allowedPortDeps,
+});
+
+const installs = {
+  rust: ports.rust({ targets: ["wasm32-wasi"], version: RUST_NIGHTLY_VERSION }),
+};
+
+ghjk.task("install-pywasi-rx", {
+  allowedPortDeps,
+  installs: [
+    ports.tar(),
+    ports.unzip(),
+  ],
+  async fn({ $ }) {
+    const WASI_VFS_DL =
+      `https://github.com/kateinoigakukun/wasi-vfs/releases/download/${WASI_VFS_VERSION}/libwasi_vfs-wasm32-unknown-unknown.zip`;
+    const LIBPYTHON_DL =
+      `https://github.com/assambar/webassembly-language-runtimes/releases/download/python%2F3.11.1%2B20230223-8a6223c/${LIBPYTHON_VERSION}.tar.gz`;
+    let WASI_SDK_DL;
+    let WASI_VFS_CLI_DL;
+    switch (Deno.build.os) {
+      case "linux":
+        WASI_SDK_DL =
+          `https://github.com/WebAssembly/wasi-sdk/releases/download/${WASI_SDK_VERSION}/${WASI_SDK_VERSION}.0-linux.tar.gz`;
+        WASI_VFS_CLI_DL =
+          `https://github.com/kateinoigakukun/wasi-vfs/releases/download/${WASI_VFS_VERSION}/wasi-vfs-cli-x86_64-unknown-linux-gnu.zip`;
+        break;
+      case "darwin":
+        WASI_SDK_DL =
```

Comment:
I wonder whether we cannot include a helper to pick the correct architecture within ghjk from the a list of artefact, wdyt?

Reply:
Yeah, definitely. A lot of the ports pull from github releases so I was thinking a general purpose github helper. Something along the lines of [eget](https://github.com/zyedidia/eget). It ought to easily allow one to specify adhoc ports inline in the ghjkfile. 

And a big part of it is making it easy to match over target triplets. Being tracked [here](https://github.com/metatypedev/ghjk/issues/7) and sort of [here](https://github.com/metatypedev/ghjk/issues/38).